### PR TITLE
HWA 11 / Gucci.com / Valid case for signing up to newsletters

### PR DESCRIPTION
**Pull request by misssEvil**
**Tested source:** gucci.com
_Valid case for signing up to newsletters_

**Scenario:**
1. Open website;
2. Accept cookie;
3. Close pop-up suggesting to change location
4. Sing Up for newsletter:

- enter test e-mail
- submit data

5. Check that pop-up notifying about successful singing up has appeared
6. close pop-up

### CODE REVIEW

**Review by EugeniaSrednitskaya — APPROVED** [review]
It's good work!

File: src/test/java/AdelinaTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
```
Comment (Belarus8500) [inline src/test/java/AdelinaTest.java:7]: no empty line needed

File: src/test/java/AdelinaTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import java.time.Instant;
+
+public class AdelinaTest extends BaseTest {
+
```
Comment (Belarus8500) [inline src/test/java/AdelinaTest.java:11]: no needed this line

File: src/test/java/AdelinaTest.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            }
+
+            WebElement popupInternational = getDriver().findElement(By.xpath("//*[@id='international-overlay']"));
+            if (popupInternational.isDisplayed()) {
+                WebElement intPopupClose = getDriver().findElement(By.className("international-overlay-close"));
+                intPopupClose.click();
+            }
+        }
+        catch (NoSuchElementException e){
+            System.out.println("Pop-up didn't appear!");
```
Comment (azellus-tercia) [inline src/test/java/AdelinaTest.java:33]: I think it could be better to replace console output.

Comment (misssEvil) [reply src/test/java/AdelinaTest.java:33]: Could you pls share any suggestions what for can I  replace it?

**Pull request closed without merging**